api/package.json

```diff
@@ -14,7 +14,7 @@
         "test:watch": "jest --watch",
         "format": "prettier --config ../.prettierrc --ignore-path .prettierignore --write .",
         "lint": "eslint src --ext .js,.ts --max-warnings 0",
-        "psql": "psql -h localhost -U postgres -d compe-plus"
+        "psql": "PGPASSWORD=postgres psql -h localhost -U postgres -d compe-plus"
```

Comment:
What does `PGPASSWORD=postgres` doe?

Reply:
I think it provides the password for the user postgres in `-U postgres` (so we don't have to type out the password everytime)

Reply:
Yep, it is setting an environment variable that psql uses to login. See [this](https://unix.stackexchange.com/questions/56444/how-do-i-set-an-environment-variable-on-the-command-line-and-have-it-appear-in-c)